docs/source/security/authentication.mdx

```diff
@@ -173,7 +194,11 @@ context: ({ req }) => {
 
```

Comment:
We need to make sure all context functions are async, it's a TS error now if they aren't.
(specifically, they need to always return a promise)

Reply:
Oh, thank you! I did a sweep to catch this in other files too :)